src/sql/src/session/vars/definitions.rs

```diff
@@ -1503,10 +1502,10 @@ pub static USER_STORAGE_MANAGED_COLLECTIONS_BATCH_DURATION: VarDefinition = VarD
     false,
 );
 
-pub static DEFAULT_NETWORK_POLICY_ALLOW_LIST: VarDefinition = VarDefinition::new_lazy(
-    "default_network_policy_allow_list",
-    lazy_value!(Vec<IpNet>; || vec![IpNet::from_str("0.0.0.0/0").expect("this is a valid IpNet")]),
-    "Network policy allow list that external user connections will be validated against.",
+pub static NETWORK_POLICY: VarDefinition = VarDefinition::new_lazy(
+    "network_policy",
```

Comment:
What do you think about naming this "fallback_network_policy" or "default_network_policy" instead of just "network_policy"? The current naming makes me think this is my current network policy, when in reality my role might have a different one applied

Reply:
My understanding of this disssions is that when we introduce the ability for users to have attached network policyes the session var will be the users `network_policy` and the system var will be the system `network_policy`, which allows us to maintain our variable inheritance model. The break is that users won't be able to change their session var value.
https://github.com/MaterializeInc/materialize/pull/29814#discussion_r1789356699

Reply:
I see, this makes sense to me! We could change this in the future if we want the session var "network_policy" to inherit from the system var "default_network_policy", but for now this LGTM